spec-grpc/src/main/java/io/a2a/grpc/utils/ProtoUtils.java

```diff
@@ -143,8 +145,8 @@ public static io.a2a.grpc.Task task(Task task) {
         public static io.a2a.grpc.Message message(Message message) {
             io.a2a.grpc.Message.Builder builder = io.a2a.grpc.Message.newBuilder();
             builder.setMessageId(message.getMessageId());
-            builder.setContextId(message.getContextId());
-            builder.setTaskId(message.getTaskId());
+            builder.setContextId(requireNonNullElse(message.getContextId(), ""));
```

Comment:
Just wondering, instead of setting the default value ourselves, should we only call setContextId if non null and then let the grpc builder set the default for us?

Reply:
Yes, makes sense :+1: